templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-backup-sh.yaml

```diff
@@ -17,18 +17,21 @@ contents:
     fi
 
     usage () {
-        echo 'Path to backup file required: ./etcd-snapshot-backup.sh ./backup.db'
+        echo 'Path to backup dir required: ./etcd-snapshot-backup.sh <path-to-backup-dir>'
         exit 1
     }
 
     ASSET_DIR=./assets
 
-    if [ "$1" != "" ]; then
-      SNAPSHOT_FILE="$1"
-    else
+    if [ -z "$1" ] || [ -f "$1" ]; then
       usage
     fi
 
+    if [ ! -d "$1" ]; then
+      mkdir -p $1
+    fi
+
+    SNAPSHOT_FILE="$1/snapshot.db"
```

Comment:
so this backing up the snapshot to the dir you passed as a param. but then the static pod assets are going into $ASSETS_DIR[1],[2]?

[1] https://github.com/openshift/machine-config-operator/pull/1336/files#diff-df2f7b2c367c15c3aa83f8b8e002b3d8R72
[2] https://github.com/openshift/machine-config-operator/pull/1336/files#diff-f553d41a05548c453f918afebe1b65e5L24

Reply:
I think this is confusing as I would expect all assets to be included in the directory path I gave. why not a single tar with all assets?

Reply:
No. Both snapshot and static pod resources go into the backup directory supplied. Oh, looks like I missed that change.